tidoop-mr-lib-api/conf/tidoop-mr-lib-api.json

```diff
@@ -1,3 +1,11 @@
 {
-  "port": 6060
+  "port": 6060,
```

Comment:
Maybe some comment lines would be appreciated about the meaning of these configuration parameters... :)